.github/workflows/push_pr.yml

```
@@ -11,21 +11,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s-version: [ "v1.15.12", "v1.16.15", "v1.17.17", "v1.18.16", "v1.19.8", "v1.20.4" ]
+        k8s-version: [ "1.23.15", "1.24.12", "1.25.8", "1.26.1" ]
```

```suggestion
        k8s-version: [ "1.23.17", "1.24.12", "1.25.8", "1.26.3" ]
```
I assume that with `k8s-version` Your intention was to refer latest release:
```
Latest Release:1.26.3 (released: 2023-03-15)
...


Minor Version
--
Minor Version         Final Patch Release          End Of Life Date
1.23                  1.23.17                      2023-02-28
```
Ref:
https://kubernetes.io/releases/
https://kubernetes.io/releases/patch-releases/#non-active-branch-history

---

@michalbiesek - I tested locally with `minikube` and it did not have support for patch version `17` on the `1.23` release. `1.24.15` was the most recent one. I think we shuold be fine here. Most managed K8s platforms are at `1.24` as their latest supported version.